dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java

```diff
@@ -44,6 +44,7 @@
  */
 public class NetUtils {
 
+    private static final Pattern MASTER_WORKER_STS_PATTERN = Pattern.compile("(master|worker)-\\d+$");
```

Comment:
Whether there are other better methods,?to judge by MASTER_WORKER_STS_PATTERN, does not seem to be a good method. NetUtils don't recommend has business mean.

Reply:
I have removed `(master|worker)` and renamed `MASTER_WORKER_STS_PATTERN` to `STS_PATTERN`.

@chenxingchun  tested on minikube in the pod named `dolphinscheduler-worker-0`, and found that the `getHost()` returned `dolphinscheduler-worker-0` which would cause network connection refused, while the correct one should be `dolphinscheduler-worker-0.dolphinscheduler-worker-headless`. So I just gave the changes.